lib/modules/manager/mise/upgradeable-tooling.ts

```diff
@@ -60,26 +60,24 @@ export const miseTooling: Record<string, ToolingDefinition> = {
     misePluginUrl: 'https://mise.jdx.dev/lang/java.html',
     config: (version) => {
       // no prefix is shorthand for openjdk
-      const versionMatch = version.match(/^(\d\S+)/)?.[1];
+      const versionMatch = /^(\d\S+)/.exec(version)?.[1];
```

Comment:
```suggestion
      const versionMatch = regEx(/^(\d\S+)/).exec(version)?.[1];
```
Same for the whole file